nuklear.h

```diff
@@ -16922,7 +16923,11 @@ nk_nonblock_begin(struct nk_context *ctx,
     } else {
         /* close the popup if user pressed outside or in the header */
         int pressed, in_body, in_header;
+#ifdef NK_BUTTON_TRIGGER_ON_RELEASE
+		pressed = nk_input_is_mouse_released(&ctx->input, NK_BUTTON_LEFT);
```

Comment:
That's weird - here I see tabs, but in `nuklear_popup.c` there are spaces :confused:.